sdks/python/apache_beam/transforms/userstate_test.py

```diff
@@ -474,14 +476,14 @@ def process(self,
         emit_timer.set(1000)
 
       @on_timer(EMIT_TIMER)
-      def emit_values(self, bag_state=beam.DoFn.StateParam(SET_STATE)):
-        for value in bag_state.read():
+      def emit_values(self, set_state=beam.DoFn.StateParam(SET_STATE)):
+        for value in set_state.read():
           yield value
-        yield 'extra'
 
       @on_timer(CLEAR_TIMER)
-      def clear_values(self, bag_state=beam.DoFn.StateParam(SET_STATE)):
-        bag_state.clear()
+      def clear_values(self, set_state=beam.DoFn.StateParam(SET_STATE)):
+        set_state.clear()
+        set_state.add('value2')
```

Comment:
Add an int, something "different" like 100, for consistency.

Reply:
I can pass some different string value instead.